--- auth.php.orig	2012-08-24 02:04:53.000000000 +0200
+++ auth.php	2012-10-19 09:18:01.000000000 +0200
@@ -598,6 +598,30 @@
     }
 
     /**
+     * Called from all enabled auth plugins that contain this method
+     * @return void
+     */
+    public function cron() {
+        global $CFG;
+
+        $this->auth_plugin_ldap();
+        if (!isset($this->config->last_cron)) {
+            set_config('last_cron', 0, $this->pluginconfig);
+        }
+
+        // notify once an hour only
+        if ($this->config->last_cron == date('Ymdh')) {
+            return;
+        }
+
+        mtrace("Starting synchronization for ldap auth plugin...");
+        $this->sync_users(true);
+        mtrace("finished.\n");
+        set_config('last_cron', date('Ymdh'), $this->pluginconfig);
+
+    }
+
+    /**
      * Syncronizes user fron external LDAP server to moodle user table
      *
      * Sync is now using username attribute.
