diff -up ./auth.php.orig ./auth.php
--- ./auth.php.orig     2012-08-24 02:04:53.000000000 +0200
+++ ./auth.php  2012-10-08 20:21:48.000000000 +0200
@@ -598,6 +598,30 @@ class auth_plugin_ldap extends auth_plug
     }
 
     /**
+     * Called from all enabled auth plugins that contain this method
+     * @return void
+     */
+    public function cron() {
+        global $CFG;
+
+       if (!isset($this->config->last_cron)) {
+           set_config('auth_ldap','last_cron', 0);
+       }
+
+       // notify once an hour only
+       if ($this->config->last_cron == date('Ymdh')) {
+           return;
+       }
+
+        print("Starting synchronization for for ldap auth plugin...");
+        $this->sync_users(true);
+        print("finished.\n");
+        
+        set_config('auth_ldap','last_cron', date('Ymdh'));
+
+    }
+
+    /**
      * Syncronizes user fron external LDAP server to moodle user table
      *
      * Sync is now using username attribute.
